src/nspawn/nspawn.c

```diff
@@ -1653,6 +1660,21 @@ static int parse_argv(int argc, char *argv[]) {
 static int verify_arguments(void) {
         int r;
 
+        SET_FLAG(arg_mount_settings, MOUNT_PRIVILEGED, arg_privileged);
+
+        if (!arg_privileged) {
+                /* machined is not accessible to unpriv clients */
+                if (arg_register) {
+                        log_notice("Automatically implying --register=no, since machined is not accessible to unprivileged clients.");
```

Comment:
This is a bit of a bummer - can't it be made to work by requiring auth via polkit?

Reply:
So we could do that. Or we could allow machined to run in user context, i.e. have a second copy of it. I think the latter would be nicer, and this has been proposed before. Becomes relevant also in context of systemd-vmspawn which after all runs unprivileged, too.

But let's focus on that in some other, later PR

Reply:
will add to TODO file